bundles/model/org.eclipse.smarthome.model.thing.tests/src/org/eclipse/smarthome/model/thing/test/hue/TestHueChannelTypeProvider.java

```diff
@@ -0,0 +1,75 @@
+package org.eclipse.smarthome.model.thing.test.hue;
+
+import java.net.URI;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+
+import org.eclipse.smarthome.core.thing.type.ChannelGroupType;
+import org.eclipse.smarthome.core.thing.type.ChannelGroupTypeUID;
+import org.eclipse.smarthome.core.thing.type.ChannelType;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeProvider;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+
+import com.google.common.collect.Lists;
+
+/**
+ *
+ * {@link TestHueChannelTypeProvider} is used for the model thing tests to provide channel types.
+ *
+ * @author Dennis Nobel - Initial contribution
+ *
+ */
+public class TestHueChannelTypeProvider implements ChannelTypeProvider {
+
+    public static final ChannelTypeUID COLORX_TEMP_CHANNEL_TYPE_UID = new ChannelTypeUID(
```

Comment:
Hm, just to be sure: Normally, a ChannelTypeUID is of the format `<bindingId>:<channelTypeId>`, am I right?